spec/lib/github_service/commands/remove_reviewer_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+RSpec.describe GithubService::Commands::RemoveReviewer do
+  subject { described_class.new(issue) }
+  let(:issue) { double(:fq_repo_name => "org/repo") }
+  let(:command_issuer) { "nickname" }
+
+  before do
+    allow(GithubService).to receive(:valid_assignee?).with("org/repo", "listed_user") { true }
```

Comment:
@europ what does the `{ true }` block at the end do here?

Reply:
@skateman .and_return(true)